src/components/folio-number-input/FolioNumberInput.vue

```diff
@@ -49,6 +51,27 @@ export default class FolioNumberInput extends Vue {
     ]
   }
 
+  /** Called when component is created. */
+  created (): void {
+    this.folioNumberString = this.folioNumber
```

Comment:
You should probably use another watcher on the prop, since this code runs only once (when this component is created), which assumes:
- the prop initially has the value you want
- the prop never changes

I don't think these are safe assumptions. Or at least, we should use a consistent code pattern for using props as v-models.

Reply:
That DatePicker looks a bit different because the prop is explicitly called "initial value".

Generally, yes, copying props to local variables in created() or mounted() isn't enough; you also need to handle props changing (ie, reactivity).